azure-nextgen-static-website/index.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+import * as cdn from "@pulumi/azure-nextgen/cdn/latest";
+import * as resources from "@pulumi/azure-nextgen/resources/latest";
+import * as storage from "@pulumi/azure-nextgen/storage/latest";
+import * as pulumi from "@pulumi/pulumi";
+import * as random from "@pulumi/random";
+
+// TODO: Remove after autonaming support is added.
+const randomSuffix = new random.RandomString("random", {
+    length: 10,
+    special: false,
+    upper: false,
+});
+
+const config = new pulumi.Config();
+const location = config.get("location") || "westus";
```

Comment:
Do we have an enum we could use here?

Reply:
I didn't see enums for location - I know the TF azure provider SDK had them...

Reply:
Azure doesn't have them. TF ones are our overlays.